src/nvim/autocmd.c

```diff
@@ -219,14 +219,14 @@ static void au_show_for_event(int group, event_T event, const char *pat)
         // show the group name, if it's not the default group
         if (ac->pat->group != AUGROUP_DEFAULT) {
           if (last_group_name == NULL) {
-            msg_puts_attr(get_deleted_augroup(), HL_ATTR(HLF_E));
+            msg_puts_hl(get_deleted_augroup(), HLF_E + 1, false);
```

Comment:
Adding `+ 1` at each call site is unfortunate. Do we want to adapt the enum and account for it elsewhere?

Reply:
I'll adjust the enum to start with 1 in a follow up refactor, this is good enough for now.